src/appleseed/renderer/modeling/frame/frame.h

```diff
@@ -102,6 +103,12 @@ class APPLESEED_DLLSYMBOL Frame
     // Access the main underlying image.
     foundation::Image& image() const;
 
+    // Access the reference image.
```

Comment:
As per my previous comment, add "Returns nullptr if there is no reference image".